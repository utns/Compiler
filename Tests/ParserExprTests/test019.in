not(d <> -(a * b + c))