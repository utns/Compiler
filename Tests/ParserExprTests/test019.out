               ____c
          ____+
                    ____b
               ____*
                    ____a
     ____-
___<>
     ____d
not
